myuw/static/js/landing.js

```diff
@@ -41,24 +41,31 @@ var Landing = {
 
     _load_desktop_cards: function() {
         Landing._reset_content_divs();
-        var desktop_body_cards = [
-            ThriveCard,
-            GradeCard,
-            FutureQuarterCardA,
-            ThankYouCard,
-            ToRegisterCard,
-            RegStatusCard,
-            SummerEFSCard,
-            SummerRegStatusCardA,
-            CriticalInfoCard,
-            InternationalStuCard,
-            SummaryScheduleCard,
-            VisualScheduleCard,
-            TextbookCard,
-            FutureSummaryScheduleCard,
-            FutureQuarterCard1,
-            SummerRegStatusCard1
-        ];
+        var desktop_body_cards;
+        if(window.user.staff_employee) {
```

Comment:
Good question. In that case, we should default to the view typical Student would see. Visual schedule on the homepage. Workday card on the Accounts page. 

@bexar feel free to chime here if you think differently or I'm mistaken.